tests/director-tests/Makefile

```diff
@@ -1 +1,51 @@
-release: ;
```

Comment:
Rename `tests/directors-tests` directory to simply `tests/director`. Other components should follow the convention.

What we have right now is against a very old approved proposal: https://github.com/kyma-project/community/blob/master/collaboration/sig-core/proposals/test-folder-consolidation.md